includes/discovery/sensors/temperature/sentry3.inc.php

```diff
@@ -1,5 +1,4 @@
 <?php
-
 $oids = snmp_walk($device, 'tempHumidSensorTempValue', '-Osqn', 'Sentry3-MIB');
```

Comment:
There are a lot of snmp_gets here, it is going to be slow.

Try this, the dd() is to see the data...
```php
$data = snmpwalk_group($device, 'tempHumidSensorTable', 'Sentry3-MIB');
dd($data);
```

Reply:
will update the diff shortly.